### PR TITLE
[PREVIEW] ROC-4402 Page objects for response to part admission/immediately payment offer

### PR DESCRIPTION
**Pull request by abshira**
### JIRA link

ROC-4402

### Change description

created page object which can be re-usable through the claimant response e2e

### Work checklist

- [ ] Unit tests added where applicable
- [ ] Route tests added for new pages
- [ ] New pages included in a11y tests
- [ ] Task list and task completeness checks updated
- [ ] Check and send page updated
- [ ] Routes, page content and page flows of new features are toggled 
- [ ] UI changes look good on mobile
- [ ] Required Google Analytics events are being sent 

### Developer self-QA run statement

- [ ] I have clicked through the running application to see if all changes I made actually work.

### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-settle-the-claim-defendant-admitted.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  yes: 'input[id=admittedyes]',
```
Comment (pragnesh-patel) [inline src/integration-test/tests/citizen/claimantResponse/pages/claimant-settle-the-claim-defendant-admitted.ts]: use optionYes , see claimant-interest.ts 
or inside fields constant define the radiobutton constant like:
```
radioButtons : {
     optionYes : 'input[id=admittedyes]',
     optionNo : ''input[id=admittedyes]'
}
```

Comment (pragnesh-patel) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-settle-the-claim-defendant-admitted.ts]: or rename radioButtons to settleClaim

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-defendant-response.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+const I: I = actor()
+
+const buttons = {
+  submit: 'input[type=submit]'
+}
+
+export class ClaimantDefendantResponsePage {
+
+  buttons (): void {
```
[the remark this answers is not in the document]
Comment (abshira) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-defendant-response.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-claim-status.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+const I: I = actor()
+
+const viewAndRespondButton = {
+  submit: 'input#button'
+}
+
+export class ClaimantClaimStatusPage {
+
+  buttons (): void {
```
[the remark this answers is not in the document]
Comment (abshira) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-claim-status.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-check-your-answers.ts [hunk 4]
```diff
@@ -0,0 +1,25 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const SubmitResponseButton = {
```
[the remark this answers is not in the document]
Comment (abshira) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-check-your-answers.ts]: Done

File: src/integration-test/tests/citizen/claimantResponse/pages/claimant-settle-the-claim-defendant-admitted.ts (hunk 1 above)
Comment (abshira) [reply src/integration-test/tests/citizen/claimantResponse/pages/claimant-settle-the-claim-defendant-admitted.ts]: Done

**Pull request merged**